code/extensions/MediaPageFrontendCreateExtension.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+class MediaPageFrontendCreateValidator extends RequiredFields {
+	public function php($data) {
+		$result = parent::php($data);
+		// maybetodo(jake): If Attribute is a required field, validate here. Need to think about how
+		// 			   to make attributes required first. (extend MediaAttribute?)
+		return $result;
```

Comment:
Yep. Not sure what the requirements of MediaPage forms will be yet. I'm considering making the validator extend itself + a bunch of default required fields in configs.